protos/mocap/mocap.proto

```diff
@@ -32,7 +33,7 @@ message VisionPositionEstimate {
     float roll = 5; // Roll angle (rad)
     float pitch = 6; // Pitch angle (rad)
     float yaw = 7; // Yaw angle (rad)
-    repeated float covariance = 8; // Row-major representation of pose 6x6 cross-covariance matrix upper right triangle
+    repeated float covariance = 8; // Row-major representation of pose 6x6 cross-covariance matrix upper right triangle. Leave empty if unknown.
```

Comment:
I think that it will [default to "0"](https://developers.google.com/protocol-buffers/docs/proto3#default) if we don't set it:

> For numeric types, the default value is zero.

I believe we should set it to `nan` if we don't want it. For instance [this example](https://github.com/mavlink/MAVSDK-Python/blob/master/examples/mission.py#L22) sets it to `nan`. A further step will be to allow for "optional" values, so that the user does not have to set it to `nan` explicitly, but we don't have that yet :blush:.

Reply:
Why not :)
I will implement that.

Reply:
Done: https://github.com/mavlink/MAVSDK-Proto/pull/105/commits/624e16b4e2213a3fd244c350236c20ed8dc45e94#diff-32ec9342f13610cc1fb904668f21d7ccR36
Backend impl: https://github.com/mavlink/MAVSDK/pull/850/commits/d1b917c27e2807f94a92f3a819c097573c204f33#diff-8d123a99eeadace7e5baafa7fa5d6105R206